anycloud/cli/src/main.rs

```diff
@@ -61,40 +71,82 @@ pub async fn main() {
       )
     );
 
-  authenticate().await;
   let matches = app.get_matches();
   match matches.subcommand() {
     ("new", Some(matches)) => {
-      let dockerfile_b64 = get_dockerfile_b64().await;
-      let app_tar_gz_b64 = get_app_tar_gz_b64(true).await;
-      let env_b64 = match matches.value_of("env-file") {
-        Some(env_file) => Some(get_env_file_b64(env_file.to_string()).await),
-        None => None,
+      let non_interactive: bool = match matches.values_of("NON_INTERACTIVE") {
+        Some(_) => true,
+        None => false,
       };
```

Comment:
nit: can be this:
```suggestion
      let non_interactive: bool = matches.values_of("NON_INTERACTIVE").is_ok();
```

Reply:
Of course!!! It would be `is_some()`, though

Reply:
ah, you're right - was still waking up :sweat_smile: